### DISCUSSION TITLE
What options works best for you?

### DISCUSSION THREAD

**Discussion by snelsi** [snelsi/lethargy-ts#2] [General]
If you think the defaults does not work well for you, please post here to tell us which devices it doesn't work well, and also which settings does work well. Many thanks!

## Options

All parameters are optional:

- `sensitivity` - Specifies the minimum value for `wheelDelta` for it to register as a valid scroll event. Because the tail of the curve has low wheelDelta values, this will stop them from registering as valid scroll events.

- `delay` - If there was a pause of this amount of milliseconds between two events, the current event is assumed to be user-triggered.

- `increasingDeltasThreshold` - If `wheelDelta` has been increasing for this amount of consecutive events, the current event is assumed to be user-triggered.

**Comment by ugljanin** [comment]
This is not working well for me. I tried with default and some custom settings, it is being registered multiple times instead of once when I scroll with trackpad on Macbook in Chrome. I tested the same with old library and there is no issue. Only one event is detected as it should. With your library, multiple events are detected.

I used this https://lethargy-ts.vercel.app/ to test again, and I can see blue dots on top of the curve and a few after that, and some at the end of the curve.

**Comment by embostan** [comment]
I tried a few settings but it makes scrolling super stutterish in Edge

It seems it detects a lot of false inertial positives

**Comment by andersonofiyai** [comment]
Defaults worked well for me on windows but I noticed for my particular project on Mac I had to increase the sensitivity. These values seemed to work great on the devices I tested on.

`import UAParser from 'ua-parser-js';
let isMac = parser.getResult().os.name.toLowerCase() === 'mac os';
const Lethargy = require('exports-loader?type=commonjs&exports=single|this.Lethargy!lethargy/lethargy');
const lethargy = new Lethargy(isMac ? 7 : 1, 50, .2);`
